icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/ULocaleTest.java

```diff
@@ -4110,6 +4110,7 @@ public void TestForLanguageTag() {
                 {"art-lojban",          "jbo",                  NOERROR},
                 {"zh-hakka",            "hak",                  NOERROR},
                 {"zh-cmn-CH",           "cmn_CH",               NOERROR},
+                {"zh-cmn-CH-u-co-pinyin", "cmn_CH@collation=pinyin", NOERROR},
```

Comment:
This random Java change looks as if it has been included in this pull request by mistake.

Reply:
Well, I included it on purpose to sync C test and Java test as far as both pass, but I can drop it from this PR and lump it together with an upcoming Java PR.